ios/AudioInputController.m

```diff
@@ -171,6 +171,28 @@ - (OSStatus) _initializeAudioGraph {
     return status;
 }
 
+- (void)storeOriginalAudioSetup
+{
+    origAudioCategory = [AVAudioSession sharedInstance].category;
+    origAudioMode =  [AVAudioSession sharedInstance].mode;
+    NSLog(@"[WebRTCVad].storeOriginalAudioSetup(): origAudioCategory=%@, origAudioMode=%@", origAudioCategory, origAudioMode);
+}
+
+- (void)restoreOriginalAudioSetup
+{
+    @try {
+        const AVAudioSession* audioSession = [AVAudioSession sharedInstance];
+        BOOL ok = [audioSession setCategory:origAudioCategory error:nil];
+
+        NSLog(@"[WebRTCVad] restore category %d", ok);
+
+        [audioSession setMode:origAudioMode error:nil];
+
+        NSLog(@"[WebRTCVad] restore mode %d", ok);
```

Comment:
is this 2nd `ok` supposed to be another value saved off from `[audioSession setMode:origAudioMode error:nil];`, or can this log message be deleted?

Reply:
Good catch. Thanks!